library/src/main/java/com/michaldrabik/classicmaterialtimepicker/model/CmtpDate.kt

```diff
@@ -0,0 +1,22 @@
+package com.michaldrabik.classicmaterialtimepicker.model
+
+import java.util.*
+
+data class CmtpDate(
+  val day: Int,
+  val month: Int,
+  val year: Int
+) {
+
+  companion object {
+    @JvmStatic
+    val calendar: Calendar = Calendar.getInstance()
```

Comment:
Can be private?

Reply:
Which field? Don't you want calendar removed?